src/test/shell/bazel/apple/bazel_objc_test.sh

```diff
@@ -159,6 +245,94 @@ EOF
     || fail "should fail to find symbol addOne"
 }
 
+# Verifies that symbols are not stripped with just `-c opt`.
+function test_symbols_not_stripped_opt() {
+  setup_objc_test_support
+
+  rm -rf ios
+  mkdir -p ios
+
+  cat >ios/main.m <<EOF
+#import <UIKit/UIKit.h>
+
+/* function declaration */
+int addOne(int num) __attribute__((noinline));
+int addOne(int num) {
+  return num + 1;
+}
+int main(int argc, char *argv[]) {
+  NSAutoreleasePool * pool = [[NSAutoreleasePool alloc] init];
+  int retVal = UIApplicationMain(argc, argv, nil, nil);
+  NSLog(@"%d", addOne(2));
+  [pool release];
+  return retVal;
+}
+EOF
+
+  cat >ios/BUILD <<EOF
+apple_binary(name = 'app',
+             deps = [':main'],
+             platform_type = 'ios')
+objc_library(name = 'main',
+             non_arc_srcs = ['main.m'])
+EOF
+
+  bazel build --verbose_failures \
+      --apple_platform_type=ios \
+      --ios_sdk_version=$IOS_SDK_VERSION \
+      --compilation_mode=opt \
+      //ios:app >$TEST_log 2>&1 || fail "should pass"
+  ls bazel-out/apl-ios_x86_64-opt/bin/ios/app_lipobin \
+    || fail "should generate lipobin (stripped binary)"
+  nm bazel-out/apl-ios_x86_64-opt/bin/ios/app_lipobin | grep addOne \
+    || fail "should fail to find symbol addOne"
```

Comment:
Is this right?  Test name seems to imply it shouldn't strip, but test names say it should?